src/components/PoolInfoCard.tsx

```diff
@@ -28,13 +28,10 @@ const InfoItem = styled(Box)(({ theme }) => ({
   display: "flex",
   alignItems: "center",
   flexDirection: "row",
-  minWidth: "50%",
+  minWidth: "100%",
```

Comment:
what happens is minWidth is deleted here? does the style break?

Reply:
I have added that property for putting 2 Item info on one row.
But in the case we put a Item in one row, We can remove `minWidth`